### PR TITLE
JBPM-5492 - Timer does not retry after failure in another node

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

**Comment by mswiderski** [comment]
depends on https://github.com/droolsjbpm/drools/pull/1050

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

File: jbpm-test-coverage/src/test/java/org/jbpm/test/functional/async/AsyncThreadContinuationTest.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ public void testRepeatFailingSyncTask() {
         Assertions.assertThat(varValues).contains("Hello Ivo asynchronously");
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 1000000)
```
Comment (krisv) [inline jbpm-test-coverage/src/test/java/org/jbpm/test/functional/async/AsyncThreadContinuationTest.java]: This probably shouldn't be committed?

Comment (krisv) [inline jbpm-test-coverage/src/test/java/org/jbpm/test/functional/async/AsyncThreadContinuationTest.java]: Apart from this, +1

Comment (mswiderski) [inline jbpm-test-coverage/src/test/java/org/jbpm/test/functional/async/AsyncThreadContinuationTest.java]: correct, it should not be, thanks

**Comment by mswiderski** [comment]
changed it back to default timeout value ... build triggered again

**Pull request merged**